### ISSUE TITLE
Missing `MVN`/`NOT` instruction

### ISSUE THREAD

**Issue by giann** [vnmakarov/mir#320]
There's not bitwise `MIR_NOT` instruction

**Comment by giann** [comment]
Maybe it's intended?
I guess i can do: `MIR_XOR value, 0xffff`

**Comment by Itay2805** [comment]
yeah you can compliment it with a XOR with FFs (I recommend using `~0` to not make mistakes). Even if intended I personally think it would be better to have it for completeness (even if the code gen can handle it)
